src/client/config/constants.js

```diff
@@ -28,3 +28,13 @@ export const MEMBER_TYPES = {
   OWNER: 'owner',
   CONTRIBUTOR: 'contributor',
 };
+
+export const LOGIN_POP_UP_WIDTH = 450;
+export const LOGIN_POP_UP_HEIGHT = 450;
+
+export const ROOT_ID = 'ROOT';
+export const TREE_VIEW_HEIGHT = 300;
+export const TREE_VIEW_MIN_WIDTH = 350;
+
```

Comment:
For the following you can have multiple errors and success codes. Try to use ranges or `response.ok`.

Reply:
These are used to set express response, I don't think I can only set `ok` but if there is a way let me know! I will rename `DEFAULT_SUCCESS_CODE` and forward the status code from the original call.

Reply:
I see. OK. In that case, you should be explicit. I would use something like this:

https://www.npmjs.com/package/http-status-codes